editor/import/resource_importer_scene.cpp

```diff
@@ -1471,6 +1471,10 @@ Ref<Animation> ResourceImporterScene::_save_animation_to_file(Ref<Animation> ani
 		}
 	}
 	anim->set_path(p_save_to_path, true); // Set path to save externally.
+	int track_count = anim->get_track_count();
```

Comment:
No need to save this in a variable, it's a cheap return and used only once

Reply:
Yes in the aspect of code style, but use `anim->get_track_count()` may make the forloop calculate this value every loop, or prevent it from loop unrolling, or it just get optimised anyway.

Reply:
But it can't be unrolled, it can't know the value in advance, no optimisation can be done here

And the value is just getting size from a map, so it's not expensive

Reply:
haha yes it's just editor code that definitely not in hot path, I will fix it!